lib/Conversion/TritonGPUToLLVM/Utility.cpp

```diff
@@ -316,6 +308,21 @@ static Value commonShflSync(Location loc, ConversionPatternRewriter &rewriter,
 
 #ifdef USE_ROCM
   GCNBuilder builder;
+
+  auto permute = [&](Value lineId, StringRef permuteInstStr) {
```

Comment:
I'm confused why this is called `lineId` instead of `laneId`?

Reply:
I think it is coming from [old code](https://github.com/ROCmSoftwarePlatform/triton/pull/391/files#diff-f641b1929956303641de2bed1e9e4019bfe5d4c4c805131f0a2b4ef3110892c9R337)

Reply:
yeah - it was lineId before